### DISCUSSION TITLE
Data links in state timeline

### DISCUSSION THREAD

**Discussion by vdwpsmt** [grafana/grafana#42656] [Dashboards, Panels & Visualizations]
data links not implemented on state timeline panel.  ( Am I right? to be confirmed)
Would be very useful for filtering or deep diving into a particular case.  Similar like you can do on a table panel.

**Comment by roygv** [comment]
I concur, there is a classic IIoT use case where machines are working on batches of "stuff".  There is a tag that represent the state that turns "true" or 1 when a batch starts. It may also have a name associated with it as it starts. State timeline is great for showing an overview of what ran (or is running) on each machine throughout time. The next "urge" is to click or drill down to the details of a specific batch on a different dashboard, while passing the batch name (or machine name/assetID) along with start and end timestamps of the batch to another dashboard to see the details

**Comment by richardachin18** [comment]
Agree that datalinks would make this already useful panel type even better. I use it to show GrafanaCLoud SyntheticMonitoring uptime check success/fail. I'd like to be able to turn the uptime check list on the left into links to the url that is being checked (i.e. the "instance" label in the uptime check metric).
![image](https://user-images.githubusercontent.com/58727954/173584907-3087f8e7-1123-4d49-9a7a-f3f0bbe5661e.png)

**Comment by natellium** [comment]
Hey there, I believe this is working since 9.1.0 :) https://github.com/grafana/grafana/pull/50226

**Comment by roygv** [comment]
Thank you! RoyOn Nov 22, 2023, at 5:23 AM, Natalia Bernarte Oses ***@***.***> wrote:﻿
Closed #42656 as resolved.
